osu.Game/Online/Multiplayer/MultiplayerClient.cs

```diff
@@ -569,6 +582,21 @@ public Task MatchEvent(MatchServerEvent e)
             return Task.CompletedTask;
         }
 
+        private void postServerShuttingDownNotification()
+        {
+            ServerShuttingDownCountdown? countdown = room?.ActiveCountdowns.OfType<ServerShuttingDownCountdown>().FirstOrDefault();
+
+            if (countdown == null)
+                return;
+
+            PostNotification?.Invoke(new SimpleNotification
```

Comment:
I'd probably make a very simple notification implementation that shows the time counting down in realtime. Likewise, I'm not sure about the server posting multiple of these. I'd have assumed it only sends one per room, then it's up to the local client to handle the display.